dev/tools/tokens/bin/gen_defaults.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Generate component theme data defaults based on the Material
+// Design Token database. These tokens were extracted into a
+// JSON file from the internal Google database.
+//
+// ## Usage
+//
+// Run this program from the root of the git repository.
+//
+// ```
+// dart dev/tools/tokens/bin/gen_defaults.dart
+// ```
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:gen_defaults/fab_template.dart';
+
+Future<void> main(List<String> args) async {
+  const String tokensDB = 'dev/tools/tokens/data/material-tokens.json';
+  final Map<String, dynamic> tokens = jsonDecode(File(tokensDB).readAsStringSync()) as Map<String, dynamic>;
```

Comment:
I don't think you need to make the type explicit here since you're already casting it

```suggestion
  final tokens = jsonDecode(File(tokensDB).readAsStringSync()) as Map<String, dynamic>;
```

Reply:
We do for the Flutter framework repo, as there is an enforced lint that needs the type declarations: 

https://dart-lang.github.io/linter/lints/always_specify_types.html